modules/social_features/social_event/social_event.module

```diff
@@ -181,7 +181,8 @@ function social_event_menu_local_tasks_alter(&$data, $route_name) {
       $node = Node::load($node);
     }
     if ($node instanceof Node) {
-      $can_show_enrollments_link = \Drupal::service('social_event.enroll')->isEnabled($node);
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.